doc/advanced-testing-mini-course/README.md

```diff
@@ -1,36 +1,32 @@
-# Advanced property-based testing mini-course
+# Property-based testing stateful systems
 
--   XXX: PBT in a nutshell?
+Property-based testing (PBT) of pure programs is an established practice by now. It’s taught in introductory university classes and it’s part of test suites in industry.
```

Comment:
Maybe have a one sentence explaining what PBT actually is? I.e generating random inputs and checking a property of the output.

Reply:
Thanks, will fix!